static/js/tools/timeline_util.ts

```diff
@@ -247,19 +266,31 @@ class TimelineParams {
     if (statsVars) {
       for (const statsVarString of statsVars.split(statsVarSep)) {
         const statsVarInfo = statsVarString.split(nodePathSep);
+        const dcid = statsVarInfo[0];
         // if statsVar path is not included in url
         // load the path from pre-built map
         if (statsVarInfo.length === 1) {
-          if (statsVarInfo[0] in statsVarPathMap) {
+          if (dcid in statsVarPathMap) {
             this.addStatsVar(
-              statsVarInfo[0],
-              statsVarPathMap[statsVarInfo[0]].map((x: number) => x.toString())
+              dcid,
+              statsVarPathMap[dcid].map((x: number) => x.toString()),
+              // TODO: Consider adding denominators to the pre-built map
+              []
             );
           } else {
-            this.addStatsVar(statsVarInfo[0], []);
+            this.addStatsVar(dcid, [], []);
           }
         } else {
-          this.addStatsVar(statsVarInfo[0], statsVarInfo.splice(1));
+          const pathsAndDenominators = statsVarInfo.splice(1);
+          this.addStatsVar(
+            dcid,
+            pathsAndDenominators.filter(
```

Comment:
can you describe these magic numbers? in particular, why 9 (it's hard to follow the format of the json, so would be good to add some documentation here. is there a better var name for elem? and should those be numbers instead of strings?)

Reply:
is this better?

Reply:
yes, better. thanks!